src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,21 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (storageDao.get(user.getLogin()).equals(null)) {
+            throw new RuntimeException("Login is exist");
+        }
+        if (user.getLogin().equals(null) || user.getPassword().equals(null)) {
+            throw new RuntimeException("Your password or login is incorrect");
+        }
+        if (user.getLogin().length() < 6 && user.getPassword().length() < 6) {
+            throw new RuntimeException("Your password and login should be bigger than 6");
```

Comment:
The logic here is incorrect. You should check the length of login and password separately, and the condition should be `length() < 6`, not combined with an `&&` operator. Also, the exception message should be specific to the field that is invalid.